content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -241,21 +241,72 @@ jobs BEFORE you reference them from another component (via the image-references
+6. PR1 and PR3 should be passing tests and ready to merge (PR2 will be failing at this point). No PRs other than PR1 and PR2 should be merged in their respective github.com repositories during the following process. In a synchronous chat with an ART team member or release manager (`@release-artists` in `#aos-art` on Slack) the following should be performed:
+   1. On the central app.ci CI cluster, a release-artist should check the current registry.ci.openshift.org image associated with the component in the `-n ocp is/4.{minor}` release image stream. For example, `oc -n ocp get istag 4.11:{old-component-name} -o=json | jq .tag.from.name` should output a pullspec like `registry.ci.openshift.org/ocp/4.11@sha256:...` for 4.11.
+   2. The release-artist should run `oc -n ocp tag {existing-registry.ci-openshift-org@sha26..} 4.{minor}:{new-component-name}` to establish a tag with the new operand component name.
+   3. Run `/retest` on PR2. Tests should now pass.
+   4. Unhold and merge PR1.
+   5. Unhold and merge PR2.
+   6. Unhold and have the release-artist merge PR3.
+   7. It is not time sensitive, but before the ART Jira ticket is closed, the release-artist must remove the old component name from CI: `oc -n ocp tag 4.{minor}:{old-component-name} -d`.
+7. Monitor the subsequent [CI nightlies on amd64 OpenShift release controller](https://amd64.ocp.releases.ci.openshift.org/#4.11.0-0.ci). Continue to do so until you see a CI nightly produced which reports the new component name in `oc adm release info <ci-nightly-release-payload-pullspec>`. If a CI nightly fails to assemble, immediately report the issue to `@team-technical-release` and `@release-artists` so that the incident can be recovered as quickly as possible.
+8. During the next working day, check [ART nightlies on the amd64 release controller](https://amd64.ocp.releases.ci.openshift.org/#4.11.0-0.nightly). Continue to do so until you see an ART nightly produced which reports the new component name in `oc adm release info <art-nightly-release-payload-pullspec>`. If an ART nightly fails to assemble, immediately report the issue to `@team-technical-release` and `@release-artists` so that the incident can be recovered as quickly as possible. 
+
+### Removing a component from the OpenShift release payload
+
+**Steps**:
+1. Submit a copy of [this ART team template](https://issues.redhat.com/browse/ART-1443) to communicate to ART that a component change is desired. Take care to mention whether the image should also be removed from the 4.x+1 branch of ART's metadata in case it has already been branched.
+2. Open and `/hold` pull request, PR1, removing references to the old component from any second level operator which includes the component in its `image-references` file.
+3. Open and `/hold` a pull request, PR2, to github.com/openshift/release which removes the `promotion:` stanza from the ci-operator configuration for the component & affected release(s).
```

Comment:
only the promotion?  Presumably there's an optional step here where you also want to remove the actual build configuration for the image, right?  (if the image is going away entirely, not just being removed from the payload)

Reply:
Added that as an option in the next push.